README.md

```diff
@@ -1,2 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Review-past-projects
-this project about code review. I select previous project to give review on it.
+
+> This is previous project about code review.
+
+
+## Built With
+
+- HTML 
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
```

Comment:
- please add (live demo link) 🔗 "because if someone checks your project live, it will be better for your project" 👍